packages/components/cypress/e2e/rating.cy.ts

```diff
@@ -0,0 +1,259 @@
+const RATING_ID = '956e063b-b40c-4fe4-bc27-53b8c4ab1e81';
+
+describe('rating', () => {
+  beforeEach(() => {
+    cy.getComponent('rating', RATING_ID);
+    cy.get('@rating').get('div.rating').as('rating-container');
+    cy.get('@rating-container').get('.star-container').as('stars');
+  });
+
+  describe('default', () => {
+    it('should render', () => {
+      cy.get('@rating').should('exist');
+    });
+
+    it('should display 5 empty stars', () => {
+      cy.get('@stars').then($stars => {
+        expect($stars.length).to.equal(5);
+
+        $stars.each((_index, star) => {
+          const classes = Cypress.$(star).attr('class').split(' ');
+          expect(classes).to.have.length(1);
+          expect(classes[0]).to.equal('star-container');
+        });
+      });
+    });
+
+    // Unfortunately the test for the hovered state does not always work.
+    //
+    // it('should change appearance on hover', () => {
+    //   cy.get('@stars')
+    //     .its('length')
+    //     .then(length => {
+    //       cy.get('@stars').each(($star, i) => {
+    //         cy.wrap($star).trigger('mouseenter');
```

Comment:
What if you try the `mouseover` event?
The `mouseover` event triggers when the mouse pointer enters an element, and its child elements. The `mouseenter` event is only triggered when the mouse pointer enters the an element, but not if it enters one of this elements child elements.

Reply:
I already tried the mouseover event, and it also doesn't work.